config/vufind/config.ini

```diff
@@ -2154,6 +2156,11 @@ treeSearchLimit = 100
 ;image_dotNoiseLevel = 100
 ;image_lineNoiseLevel = 5
 
+; Interval options:
+; Minimum interval between actions, or time from session start (seconds, default is
+; 60):
+;action_interval = 60
```

Comment:
Might there be a need to have separate values for "time from start" vs. interval? e.g. what if we want to allow somebody to do something once per hour -- then we don't want to make them wait an hour to do it the first time.

Reply:
Added!